RELEASE_NOTES.md

```diff
@@ -1,6 +1,35 @@
 # Release Notes
 
 <!-- newest release at the top please) -->
+# 2019/09/24 Prod Release Notes
```

Comment:
@smark-broad I think it's helpful to add in a section about secondary analysis even though there was no version change: 
```
Secondary Analysis
Version(s): N/A

Update subscription queries to specify latest metadata versions and trigger analysis off of ingested datasets (not just testing data).
```